sql/core/src/main/scala/org/apache/spark/sql/SQLImplicits.scala

```diff
@@ -99,33 +96,96 @@ abstract class SQLImplicits {
 
   // Seqs
 
-  /** @since 1.6.1 */
-  implicit def newIntSeqEncoder: Encoder[Seq[Int]] = ExpressionEncoder()
+  /**
+   * @since 1.6.1
+   * @deprecated use [[newIntSequenceEncoder]]
+   */
+  def newIntSeqEncoder: Encoder[Seq[Int]] = ExpressionEncoder()
```

Comment:
Wait, I'm not sure I agree... Do we want to break binary compatibility for libraries that might be using this function?  That could have even been resolved implicitly, so it would be confusing when it breaks.